src/asset-services.ts

```diff
@@ -43,23 +43,19 @@ async function load({
   manifestURL: string;
   sources: string[];
 }): Promise<void> {
-  try {
-    const response = await fetch(manifestURL);
-    const data = await response.json();
-    const entrypoints = ensureAbsoluteURLs(
-      manifestURL.substring(0, manifestURL.lastIndexOf("/") + 1),
-      data.entrypoints
-    );
-    entrypoints.concat(sources).forEach((entrypoint) => {
-      addRef(document, entrypoint);
-    });
-  } catch (error) {
-    // TODO: DE-667 - improve it, we are loosing error information here
-    // Consider using Loggly
-    // Consider applying retries
-    // Consider allowing run fallback code
-    throw new Error("Error getting assets file: " + manifestURL);
-  }
+  // TODO: DE-667 - improve error handling
+  // Consider using Loggly
+  // Consider applying retries
+  // Consider allowing run fallback code
+  const response = await fetch(manifestURL);
```

Comment:
Just curious, not seeing why is a wrong try/catch. Invalid url is not a possible scenario?

Reply:
So we can catch the error in the implementation.

Reply:
![image](https://user-images.githubusercontent.com/1157864/176516975-0ff1ae72-9c79-45d5-b6d7-6eb91507e05e.png)

I removed this catch because we were losing information.

We were catching the error but we were doing nothing with it. So, we were losing the error details and throwing a new error without details.

Now, if an error happens, this error with all of the details is thrown, so, it is better than before.

Do you agree? Am I missing something?

Reply:
Agree. Thanks for the explanation!